concourse/tasks/run-task.sh

```diff
@@ -31,26 +31,21 @@ echo "Starting task..."
 task=$(aws ecs run-task --cluster $CLUSTER \
 --task-definition $task_definition_arn --launch-type FARGATE --count 1 \
 --network-configuration $network_config \
+--started-by "Concourse" \
 --overrides '{
   "containerOverrides": [{
     "name": "'"$APPLICATION"'",
     "command": ["/bin/bash", "-c", "'"$COMMAND"'"]
   }]
 }')
 
-task_arn=$(echo $task | jq .tasks[0].taskArn)
-
+task_arn=$(echo $task | jq .tasks[0].taskArn -r)
+task_id=$(basename $task_arn)
 echo "waiting for task $task_arn to finish..."
-
-aws ecs wait tasks-stopped --tasks=[$task_arn] --cluster $CLUSTER
-
+aws ecs wait tasks-stopped --tasks="$task_id" --cluster $CLUSTER
 echo "task finished."
-
-task_results=$(aws ecs describe-tasks --tasks=[$task_arn] --cluster $CLUSTER)
-echo $task_results
-
+task_results=$(aws ecs describe-tasks --tasks=$task_id --cluster $CLUSTER)
+ecs-cli logs --cluster $CLUSTER --task-id=$task_id --since "60"
```

Comment:
can you explain why you choose 60 minutes?

Reply:
It's an arbitrary length of time that's longer than the default `since` value (0) and the length the smoke tests should realistically take.

Reply:
cool, thanks.